pkgs/servers/web-apps/cryptpad/generate.sh

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -i bash  -I nixpkgs=../../../.. -p nodePackages.node2nix nodePackages.bower2nix
+#! nix-shell -i bash  -I nixpkgs=../../../.. -p nodePackages.node2nix -p nix
+
+cryptpadSrc=$(nix eval '(import ../../../.. {}).cryptpad.src' --raw)
+echo "cryptpad src: $cryptpadSrc"
+
+nix-shell -I nixpkgs=https://github.com/nixos/nixpkgs/tarball/18.03 -p nodePackages.bower2nix --run "bower2nix $cryptpadSrc/bower.json bower-packages.nix"
```

Comment:
I'm wondering a little how long that will be available. Maybe forever since this is just a git branch, but I'm interested in knowing for sure.

Reply:
Do we really need to use a **3** year old tarball?!

Reply:
From nixpkgs dev channel, those will be available for ever, just don't receive any updates.

Reply:
Potentially node-10 is working and available on master. Dave just said he is going to have a look.

Reply:
now using nodejs-10 instead of pulling old nixpkgs

Reply:
The problem is not that they might become unavailable but that you probably need to pull multiple gigabytes to build nodejs 10 in there.

Reply:
Thank you for the info!